execution-environments/2.12-with-ansible5/bindep.txt

```diff
@@ -1,5 +1,5 @@
-python38 [platform:rpm]
-python38-devel [platform:rpm]
-python38-pip [platform:rpm]
+python39 [platform:rpm]
+python39-devel [platform:rpm]
+python39-pip [platform:rpm]
```

Comment:
This EE is outdated by now -- it was meant for ansible-core 2.12 and ansible 5 and was last published 9 months ago: https://quay.io/repository/ansible-community/test-ee?tab=tags

I am not aware if it is actually used anywhere but we're up to ansible-core 2.14 ans ansible 7 now.

We should probably leave this one alone and create a 2.14/7 one if necessary ?